deploy/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/stackrox/image-prefetcher/deploy
```

Comment:
Why it needs to be a separated module?

Reply:
As mentioned in the README, to avoid pulling all kinds of dependencies of the main binary while deploying the prefetcher.